keydata.go

```diff
@@ -151,9 +151,9 @@ const (
 	AuthModePassphrase AuthMode = 1 << iota
 )
 
-// KeyCreationData is the data required to create a new KeyData object.
+// KeyCreationParams provides parameters required to create a new KeyData object.
 // It should be produced by a platform implementation.
-type KeyCreationData struct {
+type KeyCreationParams struct {
```

Comment:
in principle this could also be just called KeyParams?

Reply:
Yeah, that's ok with me